client/data/hosting/use-cache.ts

```diff
@@ -193,6 +193,27 @@ export function useEdgeCacheDefensiveModeQuery( siteId: number | null ) {
 			} ),
 		enabled: siteId !== null,
 	} );
+
+	const { data, refetch } = query;
+	const enabled = data?.enabled ?? false;
+	const enabledUntil = data?.enabled_until ?? 0;
+
+	// Refetch the defensive mode API endpoint ten seconds after it's set to disable
```

Comment:
It's not clear what _disable_ refers to, and someone reading the code may think it relates to the action performed by the user rather than its actual outcome. I would recommend changing it to:

```suggestion
	// Refetch the defensive mode API endpoint ten seconds after it expires to
	// present the user with accurate data in the case they left their tab open
```